spotipy/client.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, auth=None, requests_session=True,
         """
         Creates a Spotify API client.
 
-        :param auth: An authorization token (optional)
+        :param auth: An authenticated OAuth2 Instance
```

Comment:
For backward-compatibility, this parameter should still be "An authorization token"